stable/concourse/values.yaml

```diff
@@ -395,6 +395,11 @@ worker:
     guardian.api.garden-server.create.failed
     unknown handle
 
+  ## Strategy for StatefulSet updates (requires Kubernetes 1.6+)
+  ## Ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset
+  ##
+  updateStrategy: OnDelete
+
```

Comment:
I can see this being convenient when set to `RollingUpdate` and tuning worker parameters. Should add an entry to the table in the README

Reply:
Sure, I can default it to `RollingUpdate` and document it.